android/java/res/layout/new_tab_page_incognito.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
Need to use the Brave license here.

Reply:
That file is 99% copied from Chromium sources. Do we replace it to our license in that case?

Reply:
I assumed that once we put it in our repo it technically becomes Brave source, but that's a good question.

Reply:
replaced it to Brave's